@voyantjs/products: dist/{catalog-policy,service-catalog-plane} ship but aren't exposed via package.json exports

## Issue

`@voyantjs/products@0.28.0` ships two server-side modules in its `dist/` that consumers need to wire the catalog plane, but neither is exposed in the package's `exports` map:

- `dist/catalog-policy.js` (+ `.d.ts`) — required to register `productCatalogPolicy` with `createFieldPolicyRegistry` in a catalog runtime.
- `dist/service-catalog-plane.js` (+ `.d.ts`) — required to call `buildProductSnapshotInput` and `createProductDocumentBuilder` from a `catalog-bridge` event subscriber.

Both are imported by `templates/operator/src/api/{catalog-bridge,lib/catalog-runtime}.ts` via:

```ts
import { productCatalogPolicy } from "@voyantjs/products/catalog-policy"
import { buildProductSnapshotInput, createProductDocumentBuilder } from "@voyantjs/products/service-catalog-plane"
```

These imports work in the workspace because `templates/operator` resolves through `workspace:*`. They fail for any consumer installing the published `@voyantjs/products@0.28.0` from npm:

```
TS2307: Cannot find module '@voyantjs/products/catalog-policy' or its corresponding type declarations.
```

Even deep imports through `dist/` don't work because the `exports` map is restrictive — Node + bundlers honor "subpath patterns must be enumerated".

## Repro

```bash
mkdir scratch && cd scratch && pnpm init -y
pnpm add @voyantjs/products@0.28.0
node --input-type=module -e "import('@voyantjs/products/catalog-policy').then(m => console.log(Object.keys(m)))"
# → ERR_PACKAGE_PATH_NOT_EXPORTED
```

## Fix

Add two entries to the `exports` block in `packages/products/package.json`:

```json
"./catalog-policy": {
  "types": "./dist/catalog-policy.d.ts",
  "import": "./dist/catalog-policy.js",
  "default": "./dist/catalog-policy.js"
},
"./service-catalog-plane": {
  "types": "./dist/service-catalog-plane.d.ts",
  "import": "./dist/service-catalog-plane.js",
  "default": "./dist/service-catalog-plane.js"
}
```

(Probably worth a one-time audit across `@voyantjs/{extras,cruises,charters,hospitality}` since the operator template imports `*/catalog-policy` from each — same gap likely exists for any vertical that's been published to npm.)

## Local mitigation

`pnpm patch @voyantjs/products@0.28.0` to add the two entries to the dist `package.json`, then `pnpm.overrides`/`patchedDependencies` in the consumer's root `package.json` to apply on install. Works but pins us to 0.28.0; we'll need to refresh the patch on every voyant bump.

--- a/.changeset/products-publish-exports-sync.md
+++ b/.changeset/products-publish-exports-sync.md
@@ -1,0 +1,19 @@
+---
+"@voyantjs/products": patch
+"@voyantjs/extras": patch
+"@voyantjs/cruises": patch
+"@voyantjs/charters": patch
+"@voyantjs/hospitality": patch
+---
+
+Fix #492: expose all workspace sub-paths in `publishConfig.exports` for vertical packages.
+
+`publishConfig.exports` (used at publish time) had drifted from the workspace `exports` map: catalog plane and content plane sub-paths shipped in `dist/` but were unreachable from the published package. Consumers installing from npm hit `ERR_PACKAGE_PATH_NOT_EXPORTED` / `TS2307` when importing them.
+
+Newly exposed sub-paths:
+
+- `@voyantjs/products`: `./catalog-policy`, `./service-catalog-plane`, `./content-shape`, `./service-content`, `./service-content-synthesizer`, `./routes-content`, `./draft-shape`
+- `@voyantjs/extras`: `./catalog-policy`, `./service-catalog-plane`, `./content-shape`, `./service-content`, `./service-content-synthesizer`, `./draft-shape`
+- `@voyantjs/cruises`: `./catalog-policy`, `./service-catalog-plane`, `./content-shape`, `./service-content-synthesizer`, `./routes-content`, `./draft-shape`
+- `@voyantjs/charters`: `./catalog-policy`, `./service-catalog-plane`, `./content-shape`, `./service-content`, `./service-content-synthesizer`, `./draft-shape`
+- `@voyantjs/hospitality`: `./catalog-policy`, `./service-catalog-plane`, `./content-shape`, `./service-content-synthesizer`, `./draft-shape`

--- a/packages/charters/package.json
+++ b/packages/charters/package.json
@@ -82,6 +82,36 @@
         "types": "./dist/adapters/index.d.ts",
         "import": "./dist/adapters/index.js",
         "default": "./dist/adapters/index.js"
+      },
+      "./catalog-policy": {
+        "types": "./dist/catalog-policy.d.ts",
+        "import": "./dist/catalog-policy.js",
+        "default": "./dist/catalog-policy.js"
+      },
+      "./service-catalog-plane": {
+        "types": "./dist/service-catalog-plane.d.ts",
+        "import": "./dist/service-catalog-plane.js",
+        "default": "./dist/service-catalog-plane.js"
+      },
+      "./content-shape": {
+        "types": "./dist/content-shape.d.ts",
+        "import": "./dist/content-shape.js",
+        "default": "./dist/content-shape.js"
+      },
+      "./service-content": {
+        "types": "./dist/service-content.d.ts",
+        "import": "./dist/service-content.js",
+        "default": "./dist/service-content.js"
+      },
+      "./service-content-synthesizer": {
+        "types": "./dist/service-content-synthesizer.d.ts",
+        "import": "./dist/service-content-synthesizer.js",
+        "default": "./dist/service-content-synthesizer.js"
+      },
+      "./draft-shape": {
+        "types": "./dist/draft-shape.d.ts",
+        "import": "./dist/draft-shape.js",
+        "default": "./dist/draft-shape.js"
       }
     },
     "main": "./dist/index.js",

--- a/packages/cruises/package.json
+++ b/packages/cruises/package.json
@@ -86,15 +86,45 @@
         "import": "./dist/adapters/index.js",
         "default": "./dist/adapters/index.js"
       },
-      "./booking-engine": {
-        "types": "./dist/booking-engine/index.d.ts",
-        "import": "./dist/booking-engine/index.js",
-        "default": "./dist/booking-engine/index.js"
+      "./catalog-policy": {
+        "types": "./dist/catalog-policy.d.ts",
+        "import": "./dist/catalog-policy.js",
+        "default": "./dist/catalog-policy.js"
+      },
+      "./service-catalog-plane": {
+        "types": "./dist/service-catalog-plane.d.ts",
+        "import": "./dist/service-catalog-plane.js",
+        "default": "./dist/service-catalog-plane.js"
+      },
+      "./content-shape": {
+        "types": "./dist/content-shape.d.ts",
+        "import": "./dist/content-shape.js",
+        "default": "./dist/content-shape.js"
       },
       "./service-content": {
         "types": "./dist/service-content.d.ts",
         "import": "./dist/service-content.js",
         "default": "./dist/service-content.js"
+      },
+      "./service-content-synthesizer": {
+        "types": "./dist/service-content-synthesizer.d.ts",
+        "import": "./dist/service-content-synthesizer.js",
+        "default": "./dist/service-content-synthesizer.js"
+      },
+      "./routes-content": {
+        "types": "./dist/routes-content.d.ts",
+        "import": "./dist/routes-content.js",
+        "default": "./dist/routes-content.js"
+      },
+      "./draft-shape": {
+        "types": "./dist/draft-shape.d.ts",
+        "import": "./dist/draft-shape.js",
+        "default": "./dist/draft-shape.js"
+      },
+      "./booking-engine": {
+        "types": "./dist/booking-engine/index.d.ts",
+        "import": "./dist/booking-engine/index.js",
+        "default": "./dist/booking-engine/index.js"
       },
       "./service-pricing": {
         "types": "./dist/service-pricing.d.ts",

--- a/packages/extras/package.json
+++ b/packages/extras/package.json
@@ -63,6 +63,36 @@
         "types": "./dist/routes.d.ts",
         "import": "./dist/routes.js",
         "default": "./dist/routes.js"
+      },
+      "./catalog-policy": {
+        "types": "./dist/catalog-policy.d.ts",
+        "import": "./dist/catalog-policy.js",
+        "default": "./dist/catalog-policy.js"
+      },
+      "./service-catalog-plane": {
+        "types": "./dist/service-catalog-plane.d.ts",
+        "import": "./dist/service-catalog-plane.js",
+        "default": "./dist/service-catalog-plane.js"
+      },
+      "./content-shape": {
+        "types": "./dist/content-shape.d.ts",
+        "import": "./dist/content-shape.js",
+        "default": "./dist/content-shape.js"
+      },
+      "./service-content": {
+        "types": "./dist/service-content.d.ts",
+        "import": "./dist/service-content.js",
+        "default": "./dist/service-content.js"
+      },
+      "./service-content-synthesizer": {
+        "types": "./dist/service-content-synthesizer.d.ts",
+        "import": "./dist/service-content-synthesizer.js",
+        "default": "./dist/service-content-synthesizer.js"
+      },
+      "./draft-shape": {
+        "types": "./dist/draft-shape.d.ts",
+        "import": "./dist/draft-shape.js",
+        "default": "./dist/draft-shape.js"
       }
     },
     "main": "./dist/index.js",

--- a/packages/hospitality/package.json
+++ b/packages/hospitality/package.json
@@ -66,15 +66,40 @@
         "import": "./dist/routes.js",
         "default": "./dist/routes.js"
       },
-      "./booking-engine": {
-        "types": "./dist/booking-engine/index.d.ts",
-        "import": "./dist/booking-engine/index.js",
-        "default": "./dist/booking-engine/index.js"
+      "./catalog-policy": {
+        "types": "./dist/catalog-policy.d.ts",
+        "import": "./dist/catalog-policy.js",
+        "default": "./dist/catalog-policy.js"
+      },
+      "./service-catalog-plane": {
+        "types": "./dist/service-catalog-plane.d.ts",
+        "import": "./dist/service-catalog-plane.js",
+        "default": "./dist/service-catalog-plane.js"
+      },
+      "./content-shape": {
+        "types": "./dist/content-shape.d.ts",
+        "import": "./dist/content-shape.js",
+        "default": "./dist/content-shape.js"
       },
       "./service-content": {
         "types": "./dist/service-content.d.ts",
         "import": "./dist/service-content.js",
         "default": "./dist/service-content.js"
+      },
+      "./service-content-synthesizer": {
+        "types": "./dist/service-content-synthesizer.d.ts",
+        "import": "./dist/service-content-synthesizer.js",
+        "default": "./dist/service-content-synthesizer.js"
+      },
+      "./draft-shape": {
+        "types": "./dist/draft-shape.d.ts",
+        "import": "./dist/draft-shape.js",
+        "default": "./dist/draft-shape.js"
+      },
+      "./booking-engine": {
+        "types": "./dist/booking-engine/index.d.ts",
+        "import": "./dist/booking-engine/index.js",
+        "default": "./dist/booking-engine/index.js"
       }
     },
     "main": "./dist/index.js",

--- a/packages/products/package.json
+++ b/packages/products/package.json
@@ -91,6 +91,41 @@
         "import": "./dist/booking-extension.js",
         "default": "./dist/booking-extension.js"
       },
+      "./catalog-policy": {
+        "types": "./dist/catalog-policy.d.ts",
+        "import": "./dist/catalog-policy.js",
+        "default": "./dist/catalog-policy.js"
+      },
+      "./service-catalog-plane": {
+        "types": "./dist/service-catalog-plane.d.ts",
+        "import": "./dist/service-catalog-plane.js",
+        "default": "./dist/service-catalog-plane.js"
+      },
+      "./content-shape": {
+        "types": "./dist/content-shape.d.ts",
+        "import": "./dist/content-shape.js",
+        "default": "./dist/content-shape.js"
+      },
+      "./service-content": {
+        "types": "./dist/service-content.d.ts",
+        "import": "./dist/service-content.js",
+        "default": "./dist/service-content.js"
+      },
+      "./service-content-synthesizer": {
+        "types": "./dist/service-content-synthesizer.d.ts",
+        "import": "./dist/service-content-synthesizer.js",
+        "default": "./dist/service-content-synthesizer.js"
+      },
+      "./routes-content": {
+        "types": "./dist/routes-content.d.ts",
+        "import": "./dist/routes-content.js",
+        "default": "./dist/routes-content.js"
+      },
+      "./draft-shape": {
+        "types": "./dist/draft-shape.d.ts",
+        "import": "./dist/draft-shape.js",
+        "default": "./dist/draft-shape.js"
+      },
       "./booking-engine": {
         "types": "./dist/booking-engine/index.d.ts",
         "import": "./dist/booking-engine/index.js",

